### DISCUSSION TITLE
SOC Alerts - Total Found: 0, but Alerts are in Kibana

### DISCUSSION THREAD

**Discussion by JoeBenutzer** [Security-Onion-Solutions/securityonion#6053] [Unsupported Versions] [answered]
I have a distributed Security Onion v2.3.80 deployment. The manager, search and sensor node were built on v2.3.0, and then ignored for quite some time during other projects. I was able to get back to it recently, ran soup to get to v2.3.80 and installed 3 more sensors (from ISO). However, when I went to the Security Onion Console and checked Alerts, there were "Total Found: 0". I believe this was working months ago, but couldn't swear to it. Checking Last 12 months shows me 0 Found as well. Kibana shows the alerts just fine. They're just not getting from there to the Console, and I'm unsure of the mechanism at to how that actually happens.

Grid status is OK for all
so-status shows OK for all
salt-call state.highstate on manager shows 1 failure (There is an agent [Wazuh] with the same name)
Nothing in any log file under /opt/so/log leaps out at me

In the searching I've done, I've seen a couple of others with the same problem, but they were either unanswered, or resolved by rebuilding the system, (which I hesitate to do, without putting effort into fixing first). Where should I look?

**Comment by tiger-greenhell** [comment, marked as answer]
#5920

**Reply by JoeBenutzer** [reply]
That was it. Thank you very much!
